Common/src/main/java/com/agilepro/commons/models/customer/UserSettingModel.java

```diff
@@ -0,0 +1,119 @@
+package com.agilepro.commons.models.customer;
+
+import com.yukthi.webutils.common.annotations.Model;
+import com.yukthi.webutils.common.annotations.NonDisplayable;
+
+/**
+ * The Class UserSettingModel.
+ * 
+ * @author Pritam
+ */
+@Model(name = "UserSetting")
+public class UserSettingModel
+{
+	/**
+	 * The id.
+	 **/
+	@NonDisplayable
+	private Long id;
+
+	/**
+	 * The user id.
+	 **/
+	private Long userId;
+
+	/**
+	 * The project id.
+	 **/
+	private Long projectId;

```

Comment:
User setting is suppose to be generic. Today it is used for project id, tomorrow it can be something else. 

So instead of project id it should be value (of type string). And there should be key. So that this will behave like a m,ap.